lib/app/modules/us_core_r4/us_core_device_sequence.rb

```diff
@@ -145,9 +145,35 @@ def validate_resource_item(resource, property, value)
         validate_history_reply(@device, versioned_resource_class('Device'))
       end
 
-      test 'Device resources associated with Patient conform to Argonaut profiles' do
+      test 'Demonstrates that the server can supply must supported elements' do
         metadata do
           id '07'
+          link 'https://build.fhir.org/ig/HL7/US-Core-R4/general-guidance.html/#must-support'
+          desc %(
+          )
+          versions :r4
+        end
+
+        must_support_elements = [
+          'Device.udiCarrier',
+          'Device.udiCarrier.carrierAIDC',
+          'Device.udiCarrier.carrierHRF',
+          'Device.type',
+          'Device.patient'
+        ]
+        must_support_elements.each do |path|
+          truncated_path = path.gsub('Device.', '')
+          already_found = @instance.must_support_confirmed.include?(path)
+          element_found = already_found || can_resolve_path(@device, truncated_path)
```

Comment:
It appears that we are just checking a single device, even if they returned multiple devices from the searches.  I think we should look across all devices provided for the patient, and pass them if at least one has a must support element.

Reply:
I made it check all the resources in the reply. We should probably make it check all the resources for the other tests as well... but I think that should be another task.

Reply:
What do you mean by 'the other tests'?